misc/userscripts/qute-pass

```diff
@@ -58,6 +60,8 @@ argument_parser.add_argument('url', nargs='?', default=os.getenv('QUTE_URL'))
 argument_parser.add_argument('--password-store', '-p',
                              default=os.getenv('PASSWORD_STORE_DIR', default=os.path.expanduser('~/.password-store')),
                              help='Path to your pass password-store')
```

Comment:
Can you add `(only used in pass-mode)` to the help text?

Reply:
Good catch, added. :+1: